files/halite_gem/poise_python/python_providers/portable_pypy3.rb

```diff
@@ -27,26 +27,11 @@ class PortablePyPy3 < Base
       include PoiseLanguages::Static(
         name: 'pypy3',
         # Don't put prereleases first so they aren't used for prefix matches on ''.
-        versions: %w{2.4 5.7.1-beta 5.7-beta 5.5-alpha-20161014 5.5-alpha-20161013 5.2-alpha-20160602 2.3.1},
-        machines: %w{linux-i686 linux-x86_64},
-        url: 'https://bitbucket.org/squeaky/portable-pypy/downloads/pypy3-%{version}-%{kernel}_%{machine}-portable.tar.bz2'
+        versions: %w{3.6-7.2.0},
+        machines: %w{linux-x86_64},
+        url: 'https://github.com/squeaky-pl/portable-pypy/releases/download/pypy%{version}/pypy%{version}-%{kernel}_%{machine}-portable.tar.bz2'
       )
```

Comment:
While you're at it you may need to consider adding the checksum like in PortablePyPy.